src/tests/index.test.js

```diff
@@ -425,8 +425,78 @@ describe('Evolv client integration tests', () => {
       expect(evolv.context.get('newProfileInfo')).equals('this');
     });
 
+    it('should put in appropriate confirmations', async () => {
+      const environment = '579b106c73';
+      const endpoint = 'https://participants-frazer.evolv.ai/';
```

Comment:
would get rid of old emplyee names in the test!

Reply:
I figured the experiments were all configured there, and all the other tests use the same endpoint. I can try it without it though

Reply:
of course it works since we're mocking all the responses 🤦